File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        // brute force
        int[] result = new int[2];
        for (int i = 0; i < nums.length; i++) {
            for (int j = 0; j < nums.length; j++) {
                result[0] = i;
                result[1] = j;
                System.out.println(result);
                if (i != j && nums[i] + nums[j] == target) {
                    return result;
                }
            }
        }
        return null;
    }
}

File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        int l = 0;
        int r = numbers.size() - 1;
        while (l < r) {
            int total = numbers[l] + numbers[r];
            if (total == target) {
                break;
            } else if (total > target) {
                r--;
            } else {
                l++;
            }
        }
        // vector<int> result({l + 1, r + 1});
        return {l + 1, r + 1};
    }
};
